tests/Agent/IntegrationTests/SharedApplications/Common/MultiFunctionApplicationHelpers/NetStandardLibraries/LogInstrumentation/SerilogLoggingAdapter.cs

```diff
@@ -86,15 +86,18 @@ public void ConfigurePatternLayoutAppenderForDecoration()
             _log = loggerConfig.CreateLogger();
         }
 
+        // net462 is using Serilgo 1.5.14 - a very old, but supported version of Serilog. This does not have a Console sink that supports Json.
         public void ConfigureJsonLayoutAppenderForDecoration()
         {
+#if !NET462
```

Comment:
Since this isn't really tied to 4.6.2, maybe it would be a little cleaner to create a `SERILOG_V1` define, like we do with RabbitMQ? https://github.com/newrelic/newrelic-dotnet-agent/blob/12c867c86e850b974dc5686bd5224aef4298d89e/tests/Agent/IntegrationTests/SharedApplications/Common/MultiFunctionApplicationHelpers/NetStandardLibraries/RabbitMQ.cs#L6

Reply:
I tend to dislike custom defines since it abstracts away the framework.  I like to know the condition is really checking.